src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -84,7 +85,8 @@ object WdlExpression {
   }
 }
 
-case class WdlExpression(ast: AstNode) {
+case class WdlExpression(ast: AstNode) extends WdlValue {
+  val wdlType = WdlExpressionType
```

Comment:
Should add an override.

Also would it make sense to make this `type wdlType` instead of `val wdlType`?